spot-client/config.js

```diff
@@ -139,6 +139,9 @@ window.JitsiMeetSpotConfig = {
             domain: process.env.XMPP_HOSTS_DOMAIN || 'meet.jit.si',
             muc: process.env.XMPP_HOSTS_MUC_URL
                 || 'conference.meet.jit.si'
+        },
+        p2p: {
+            useStunTurn: true
```

Comment:
We'll have to update the deployment config too (when we're about to deploy) because right now with the current setup the deployment config overrides the config completely.

Reply:
oh okay

Reply:
will this work better if places in the defaults then ?

Reply:
My other PR about config refactoring could help but I want to make sure we're in agreement on that one. But yeah, I think right now adding default true would outside of config would be really good.